### ISSUE TITLE
[Feature] AddFeatureShortDescriptionHere

### ISSUE THREAD

**Issue by jariterbang** [NPBruce/valkyrie#1572]
## Is your feature request related to a problem? Please describe.**
Not really. I'm working on localization to Bahasa Indonesia but it's not supported yet, so I just translate a scenario in the EN file, and upload it as a NEW ISSUE.

## Describe the solution you'd like
Please add an option for Bahasa Indonesia language (ID). thank you.

## Describe alternatives you've considered
that's it.

## Additional context
At first it's just for playing with my family but I want to try introduce valkyrie to my community here.

**Comment by jariterbang** [comment]
thank you very much for the kind reply, I will gladly provide some assistance with the translation as best as I can. @vidimus @antontimmermans

**Comment by vidimus** [comment]
@jariterbang Here is the text from the official app. You will need to translate it to provide Indonesian language support for Valkyrie, if possible. Please contact first with one of the developers to see if adding Indonesian language is possible or not before starting the translation.

[Mansions of Madness 2ed - Text from the app.xlsx](https://github.com/NPBruce/valkyrie/files/9374232/Mansions.of.Madness.2ed.-.Text.from.the.app.xlsx)
